pkg/gateways/http/transfer/create_transfer.go

```diff
@@ -14,19 +14,32 @@ import (
 func (s *Handler) CreateTransfer(w http.ResponseWriter, r *http.Request) {
 	var requestBody TransfersRequest
 	err := json.NewDecoder(r.Body).Decode(&requestBody)
-	token := r.Header.Get("Authorization")
-
 	if err != nil {
 		w.WriteHeader(http.StatusBadRequest)
 		return
 	}
+	header := r.Header.Get("Authorization")
+
+	token, err := CheckCredential(header)
 
 	l := s.logger.WithFields(log.Fields{
 		"module": "https",
 		"method": "processTransfer",
 		"token":  token,
```

Comment:
taking a look now, i'm pretty sure it's not a good idea to log the user token =(